src/euphorie/client/browser/risk.py

```diff
@@ -503,11 +503,20 @@ def calculatePriority(self, risk, reply):
 
 class ImageUpload(BrowserView):
 
+    def _return(self):
```

Comment:
Maybe a name like `redirect` or `_redirect` would be better, but this is a really minor thing

Reply:
Just in case you like consistency:
https://github.com/euphorie/Euphorie/blob/d81c67d679d0eaf5dd08583579a172f1bc99a5db/src/euphorie/client/browser/reset_password.py#L51-L56

Reply:
Thanks! I knew we already had something like that, and I also did not like the name `_return` either... I this case, I like consistency a lot, will change! :-)